README.md

```diff
@@ -34,8 +34,8 @@ Release details and process are found in [Devfile Release](RELEASE.md)
 
 ## How to build
```

Comment:
this section is duplicating the content in contributing doc. We shouldn't spent effort to maintain same content separate doc. Please use the link to contributing.md, e.g. `./CONTRIBUTING.md#building`

Reply:
I agree about the duplication :) Thinking that in general the contribution guidelines are more related to information about how someone could open a PR or an issue, I was wondering if the `contributing.md` is the right file to maintain this section? WDYT?

Reply:
Thank you for pointing that out! To remove the duplication I agree with contributing.md probably not being the right spot for the build instructions, I'll remove it from contributing and keep in in the `readme.md` only

Reply:
@yangcao77 @thepetk please see the latest commit for my updated changes.

Reply:
should directly link to the `building` section. not the top of the contributing doc